examples/js/effects/VREffect.js

```diff
@@ -11,14 +11,18 @@
 
 THREE.VREffect = function ( renderer, onError ) {
 
+	var scope = this;
+
 	var vrHMD;
 	var isDeprecatedAPI = false;
 	var eyeTranslationL = new THREE.Vector3();
 	var eyeTranslationR = new THREE.Vector3();
 	var renderRectL, renderRectR;
 	var eyeFOVL, eyeFOVR;
 
-	function gotVRDevices( devices ) {
+	function gotVRDevices ( devices ) {
+
+		scope.displays = devices;
```

Comment:
This is sneaky... I would, instead, define a local variable `displays` and add a `.getDisplays()` public method.